### ISSUE TITLE
[UI] Starting of localization for Olive version 0.2

### ISSUE THREAD

**Issue by pafri** [olive-editor/olive#1313]
**Commit Hash** 

**Platform** 

**Summary**
Localization of graphical user interface: lacking of the file.

**Additional Information / Output**
Unlike version 0.1 I don't see/can't find the localization folder with files, named for example Translation etc.

**Comment by elsandosgrande** [comment]
@itsmattkc Would you say that Olive Razor is in good-enough shape to start translating the user interface?

**Comment by itsmattkc** [comment]
I wouldn't recommend it. People could start submitting, but since things are still changing a lot, there's a good chance they'll break and need to keep updating the translations pretty regularly. I guess if people PR them, I'd consider merging them.

**Comment by elsandosgrande** [comment]
I can't speak for others, but I wouldn't have much of a problem with that, though in my case that's because it will take much more effort to complete the translation in the first place than it will to keep them up to date, thus making the constant changes a non-issue.

By the way, how does one add translation files now that Qmake is not supported? Does CMake do it automatically?

**Comment by samytichadou** [comment]
Hey i'm down to help with french translation whenever you guys think olive is in good-enough shape for it ! Replying here to get the news :) 
And same here, i don't mind updating them now and then, cheers !

**Comment by elsandosgrande** [comment]
@samytichadou From what I gather, changes are not made now and again, but very frequently.

**Comment by samytichadou** [comment]
All right then, when you guys think it is a good timing to start the translation, poke me and i'll try to keep it up with the changes ! If it can be useful, being a non-coder, i would be glad to be of some help with olive !

**Comment by pafri** [comment]
Quick updates of strings aren't problem for me. The last time, when I've translated Olive (0.1), it was made using QtLinguist (.ts file). Also I would guess, that the work made on translation is reusable (translation memory in localization editors, or for example .qph library(ies) for QtLinguist made from the last translation for further use), even if the text is moved somewhere else "in the structure" or partly/changed). It is done automatically. Even I could do it manually in terminal (lupdate command) - if there would be the specification for it in .pro file, or so (if there is any pro file in source code. The times changed, I'd guess :-)? It's quite long time, when I had had to make something like this manually. So I am waiting for .po or .ts file for the work to be done.
